sdks/java/core/src/main/java/org/apache/beam/sdk/values/BeamRecordType.java

```diff
@@ -49,22 +49,22 @@ public void validateValueType(int index, Object fieldValue)
     * Get the coder for {@link BeamRecordCoder}.
     */
    public BeamRecordCoder getRecordCoder(){
-     return BeamRecordCoder.of(this, fieldsCoder);
+     return BeamRecordCoder.of(this, fieldCoders);
    }
 
-   public List<String> getFieldsName(){
-     return fieldsName;
+   public List<String> getFieldNames(){
```

Comment:
But BeamSqlRecordType doesn't override getFieldByIndex to return name + type, it still just returns the name, doesn't it? It sounds like the existing methods that just talk about fields are the ones that should probably be renamed to talk about field names. What do you think?